core/services/directrequest/delegate.go

```diff
@@ -92,7 +92,7 @@ func (d *Delegate) ServicesForSpec(jb job.Job, qopts ...pg.QOpt) ([]job.ServiceC
 		)
 
 	logListener := &listener{
-		logger:                   svcLogger.Named("DirectRequest"),
+		logger:                   svcLogger.Named("Listener"),
```

Comment:
maybe DirectRequestListener?

Reply:
It is already scoped as DirectRequest from the parent logger: `DirectRequest.Listener`